README.md

```diff
@@ -82,7 +82,17 @@ Documentation search is powered by [Algolia's DocSearch](https://community.algol
 
 ## Contributing
 
-Please read through our [contributing guidelines](https://github.com/twbs/bootstrap/blob/master/CONTRIBUTING.md). Included are directions for opening issues, coding standards, and notes on development.
+Commit messages should follow semantic versioning.
+
+```
+breaking: (major breaking change - e.g. finally removing float in favor of hovering)
```

Comment:
Also includes changing or removing class names or functionality, and in our case many changes to documentation that break parity with mainline bootstrap.